OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,57 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+struct dynamically_sized_perf_event_callchain_sample {
+  struct dynamically_sized_perf_event_callchain {
+    uint64_t nr;
+    std::unique_ptr<uint64_t[]> ips;
+
+    explicit dynamically_sized_perf_event_callchain(uint64_t dyn_size)
+        : nr{dyn_size}, ips{make_unique_for_overwrite<uint64_t[]>(dyn_size)} {}
+  };
+
+  perf_event_header header;
+  perf_event_sample_id_tid_time_streamid_cpu sample_id;
+  dynamically_sized_perf_event_callchain call_chain;
+
+  explicit dynamically_sized_perf_event_callchain_sample(uint64_t dyn_size)
+      : call_chain{dyn_size} {}
+};
+
+class SampleCallchainPerfEvent : public PerfEvent {
```

Comment:
This should be `CallchainSamplePerfEvent` as "callchain" acts as an adjective for sample.

Reply:
I'm fine with that.